setup.cfg

```diff
@@ -0,0 +1,6 @@
+[tool:pytest]
+addopts =
+    tests
+    --cov plugins
+    --cov-fail-under=100
```

Comment:
These should be moved into `[coverage:run]` and `[coverage:report]` .  See coala/coala `setup.cfg`